commerce_coordinator/apps/rollout/utils.py

```diff
@@ -0,0 +1,17 @@
+""" Rollout Utility Functions"""
+
+import re
+
+
+def is_legacy_order(order_number: str) -> bool:
+    """ Determine if an order is a Legacy Ecommerce order """
+    if not order_number:
+        return False
+    return re.search("EDX-[0-9]{1,6}", order_number) is not None
+
+
+def is_uuid(value: str) -> bool:
+    """ Determine if an value is a UUID """
+    if not value:
+        return False
+    return re.search(r"\b[A-Fa-f0-9]{8}(?:-[A-Fa-f0-9]{4}){3}-[A-Fa-f0-9]{12}\b", value) is not None
```

Comment:
We can check for a valid UUID by initializing its object:

```suggestion
    from uuid import UUID
    is_valid = True
    try:
        uuid_obj = UUID(uuid_to_test)
    except ValueError:
        is_valid=False
    return is_valid
```

Reply:
This seemed like more work... especially since i had the regex handy...